### PR TITLE
change eruption to use new 3-monitor setup

### PR DESCRIPTION
**Pull request by cg505**
This is currently running on eruption. Unfortunately we have to get rid of the panels spanning monitors change because it's totally broken.

### CODE REVIEW

File: modules/ocf_desktop/files/xsession/fix-displays [hunk 1]
```diff
@@ -54,11 +54,26 @@ if __name__ == '__main__':
     else:
         sorted_monitors = list(sorted(monitors))
 
-    prev = None
-    for monitor in sorted_monitors:
-        if prev:
-            subprocess.check_call(['xrandr', '--output', monitor, '--right-of', prev])
-        prev = monitor
+    # special case for eruption: right monitor is rotated left 90 degrees
+    if len(monitors) == 3 and hostname == 'eruption':
```
Comment (ja5087) [inline modules/ocf_desktop/files/xsession/fix-displays]: I'm not a fan of encoding special cases in the program like this. Is there a way to encode this info per host that the program could fetch? or even some kind of configuration file that overrides specifically for eruption.

Comment (dkess) [reply modules/ocf_desktop/files/xsession/fix-displays]: Could be worth looking into something like [autorandr](https://github.com/phillipberndt/autorandr) (which is in the Debian Buster repos) to dynamically set the monitor configs.

Comment (cg505) [reply modules/ocf_desktop/files/xsession/fix-displays]: autorandr looks nice

**Comment by cg505** [comment]
opened #834 to track autorandr and other improvements. I'm gonna merge this right now because that shouldn't block eruption working correctly.

**Pull request merged**